package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kolibri-design-system",
-  "version": "v1.3.0",
+  "version": "3.0.0",
```

Comment:
I also removed `v` because I had an impression it shouldn't be there, now that we're going to install KDS as

```
"dependencies": {
  "kolibri-design-system": "^3.0.0",
}

https://docs.npmjs.com/about-semantic-versioning#using-semantic-versioning-to-specify-update-types-your-package-can-accept

Reply:
Yes, this seems correct!

Might be good to create a small utility at some point to check that the tag matches the version, like we do for Kolibri python versions.